components/partials/banners/welcome.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <v-card
     v-if="
-      !hasMadeAnyTransfers && !hasAnyTradingAccountBalances && status.isIdle()
+      ((!hasMadeAnyTransfers && !hasAnyTradingAccountBalances) ||
```

Comment:
Let's extract this into a separate computed property please, too many conditionals here.